testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -4108,6 +4108,19 @@ static void checkRlikeFails(SqlOperatorFixture f) {
     f.checkNull("CHARACTER_LENGTH(cast(null as varchar(1)))");
   }
 
+  @Test void testLenFunc() {
+    final SqlOperatorFixture f0 = fixture().setFor(SqlLibraryOperators.LENGTH);
+    f0.checkFails("^length('hello')^",
+        "No match found for function signature LENGTH\\(<CHARACTER>\\)",
+        false);
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.SNOWFLAKE);
+    f.checkScalar("length('hello')", "5", "INTEGER NOT NULL");
+    f.checkScalar("length('')", "0", "INTEGER NOT NULL");
+    f.checkScalar("length(CAST('x' as CHAR(3)))", "3", "INTEGER NOT NULL");
+    f.checkScalar("length(CAST('x' as VARCHAR(4)))", "1", "INTEGER NOT NULL");
+    f.checkNull("length(CAST(NULL as CHAR(5)))");
```

Comment:
None of these tests has a call to 'len(...)'. Maybe LEN is getting tested via some aliasing mechanism. It's difficult to tell. 

So, let's have at least one test for LEN. So we can still trust the test even if the clever stuff fails.

Reply:
I just forgot to change them, fixed now